### ISSUE TITLE
(Interactive Description) Combined balloon description at wall

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#520]
While reviewing #490, I noticed that when 2 balloons are in play and against the wall, I get very good combined balloon and induced charge effect descriptions as long as one balloon is grabbed.

But as soon as I both balloons are released, I no longer get the combined "balloons" describe with the combined induced charge effect. I just get a single balloon description with the combined induced charge effect.

When Yellow Balloon is grabbed with a few charges, against wall AND next to Green Balloon. The Context Response and the Wall's dynamic State Description correctly describe the combined induced effect:
![Screen Shot 2021-07-27 at 11 49 11 AM](https://user-images.githubusercontent.com/1902116/127171225-5c19defd-f63b-4b6e-bad3-56884f451f74.png)

However, as soon as I release the Yellow Balloon, I get 2 responses
![Screen Shot 2021-07-27 at 11 49 26 AM](https://user-images.githubusercontent.com/1902116/127174481-ebfc9282-d5ff-4ab2-9193-997f853d2008.png)
- "Released" (a release response, which is correct) 
and then a  response about the Yellow Balloon that implies that it alone is creating the induced charge effect which is actually a combined induced charge effect.
- "Yellow Balloon, no change in position. Yellow Balloon, at wall, next to green balloon. Negative charges in wall move away from Yellow Balloon a lot. Positive charges do not move."

@jessegreenberg, I know we have worked on this edge case before, so I am not sure if this is new, or if we just decided to do ignore the combined effect on release, or if we missed it.

Regardless, since we are monitoring for a combined induced charge effect when the balloons are active and considered to be next to each other, is there a quick fix for this situation when both balloons are released? For example, is it possible to hear "balloons" rather than "Yellow balloon" alone in the last response (above) after the yellow balloon is released?

- "Yellow Balloon, no change in position. Yellow Balloon, at wall, next to green balloon. Negative charges in wall move away from **balloons** a lot. Positive charges do not move."

**Comment by jessegreenberg** [comment]
We discussed this today at 7/27/21 - 

We identified a bug that was preventing the combinded description from being used. Should be fixed in the above commit. @terracoda would you like to verify?

**Comment by terracoda** [comment]
@jessegreenberg , using PhET Test, I am still seeing some inconsistencies when the balloons are overlapped. Sometimes only the effect of a single balloon is described in the charge movement string.

In the screen shot below, only the green balloon is described as having an effect. This might be because the YB only has 6 charges, and the GB has "many" but I think 6 charges still has a "a little bit" of  an effect on their own.
<img width="427" alt="Screen Shot 2021-08-02 at 9 54 15 AM" src="https://user-images.githubusercontent.com/1902116/127861537-7e8f4021-9841-412c-b41f-0bc994655ac7.png">

 I did another test. It takes 6 charges to make the Yellow balloon actually stick to the wall. So with 6 charges (i.e. "a few") on the YB, and "several" on the GB, I seem to always get the "balloons" as having an induced charge effect when the GB only has "several" rather than "many" more negative charges. 

@jessegreenberg, is this a bug - this difference between "several" and "many"? Can you fine tune the model a little bit more to make sure when a balloon has a enough charge to make it stick to the wall that a combined effect is described when the balloons are "next to" each other?

**And noting other behaviour:**
The Wall's induced charge can change between "a little bit" and "quite a lot" of movement depending on how much the balloons overlap and how much negative net charge the balloons have. I think the range in described movement is the intended behaviour. Does that make sense to you, @jessegreenberg?

When the balloons are not described as "next to" each other, only the movement of that single balloon is described. This is the intended behaviour.

**Comment by terracoda** [comment]
If this fine-tuning is hard, let's quickly discuss. Maybe it's something we can't always capture?

**Comment by jessegreenberg** [comment]
I tried to reproduce the inconsistencies you mentioned @terracoda but could not. I tried to recreate the same case where the YB has 6 charges:
![image](https://user-images.githubusercontent.com/6396244/128574550-fc9c4954-7511-4682-b400-df954fd7bda3.png)

For both the movement and release alerts it combined them. Is there anything else I am missing while testing?

**Comment by terracoda** [comment]
@jessegreenberg, I can't reproduce now either. Let's chalk it up to glitch and move on :-)

**Comment by jessegreenberg** [comment]
Thanks for checking! Very possible changes since https://github.com/phetsims/balloons-and-static-electricity/issues/520#issuecomment-891022918 have resolved this naturally.
